indico/migrations/versions/20240130_1531_17996ef18cb9_update_constraint_valid_enum_personal_data_type.py

```diff
@@ -0,0 +1,29 @@
+"""Update constraint valid_enum_personal_data_type
```

Comment:
```suggestion
"""Add picture personal data type for registrations
```

and I'd rename the file to `20240130_1531_17996ef18cb9_add_picture_personal_data_type_registrations.py`

Context: When looking for an alembic revision I usually search in the filenames for something I know it touched. In this case it'd be "registration" and "picture".

Also, that title is nicer IMHO ;)